server/src/main/kotlin/eu/kanade/tachiyomi/network/interceptor/Chromium.kt

```diff
@@ -0,0 +1,58 @@
+package eu.kanade.tachiyomi.network.interceptor
```

Comment:
looks like we should put playwright code out of this package, its interfering with the rule of one interceptor per file.